modules/migration-debugging-velero-resources.adoc

```diff
@@ -61,6 +61,25 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
   backup describe 0e44ae00-5dc3-11eb-9ca8-df7e5254778b-2d8ql
 ----
 
+The following types of restore errors and warnings are shown in the output of a `velero describe` request:
+
+* `Velero`: A list of messages related to the operation of Velero itself (for example, messages related to connecting to the cloud, reading a backup file, etc.)
+* `Cluster`: A list of messages related to backup or restore of cluster-scoped resources
+* `Namespaces`: A list of list of messages related to backup or restore namespace-scoped resources
+
+One or more errors in any of the above categories causes a `Restore` operation to have a status of `PartiallyFailed`, rather than `Completed`. Warnings do not cause a change to the completion status.
+
+[NOTE]
+====
+For resource-specific errors, that is, `Cluster` and `Namespaces` errors, the `restore describe --details` output includes a resource list that lists all resources that Velero succeeded in restoring. For any resource that has such an error, check to see if the resource is actually in the cluster.
+====
+
+[NOTE]
+====
+If there are `Velero` errors, but no resource-specific errors, in the output of a `describe` command, it is possible that the restore completed without any actual problems in restoring workloads, but carefully validate post-restore applications.
+For example, if the output contains `PodVolumeRestore` or Node Agent-related errors, check the status of `PodVolumeRestores` and `DataDownloads`. If none of these are failed or still running, then volume data might have been fully restored.
+====
+
```

Comment:
```suggestion
[IMPORTANT]
====
* For resource-specific errors, such as `Cluster` and `Namespaces` errors, the `restore describe --details` output contains a resource list that lists all resources that Velero was able to restore successfully. For each resource that shows such an error, check whether the resource is actually in the cluster.

* If the output of a description command contains `Velero` errors but no resource-specific errors, it is possible that the restore completed without any actual workload restore issues, but check the applications carefully after restoration.
For example, if the output contains `PodVolumeRestore` or Node Agent related errors, check the status of `PodVolumeRestores` and `DataDownloads`. If none of these applications have failed or are still running, the volume data might have been fully restored.
====
```

Reply:
Clubbing the two seprate notes together. Since, there is action involved, I would suggest changing it into an "Important" notice.